Exercicio2/src/A/imperativo/2a.ts

```diff
@@ -0,0 +1,19 @@
+let list: Array<Object> = [
```

Comment:
a programação imperativa, seria mais como usar um for no lugar do filter que você fez ali. No caso do seu código